consensus/misc/eip1559.go

```diff
@@ -65,16 +78,9 @@ func CalcBaseFee(config *params.ChainConfig, parent *types.Header, parentL1BaseF
 		return big.NewInt(10000000) // 0.01 Gwei
 	}
 
-	scalar := config.Scroll.BaseFeeScalar
-	if scalar == nil {
-		scalar = DefaultBaseFeeScalar
-	}
-	overhead := config.Scroll.BaseFeeOverhead
-	if overhead == nil {
-		overhead = DefaultBaseFeeOverhead
-	}
-
-	return calcBaseFee(scalar, overhead, parentL1BaseFee)
+	lock.RLock()
```

Comment:
What about moving this read lock into `calcBaseFee` function?

Reply:
`MinBaseFee` does not use these, it uses the passed in arguments.

Reply:
thanks. The current position of read lock is correct.